### PR TITLE
Power node description fix.

### PR DESCRIPTION
**Pull request by JakeDaBoss**

### CODE REVIEW

**Review by NanakoAC — APPROVED** [review]

File: html/changelogs/TheLion1675 - Spellcheck.yml [hunk 1]
```diff
@@ -34,3 +34,4 @@ delete-after: True
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
   - spellcheck: "Fixed a spelling error in Containment round description."
```
Comment (NanakoAC) [inline html/changelogs/TheLion1675 - Spellcheck.yml]: It's bad form to edit changelogs created by other PRs, they may not be existing by the time this is merged

**Review by NanakoAC — CHANGES_REQUESTED** [review]
changelog thing

File: html/changelogs/TheLion1675 - Spellcheck.yml (hunk 1 above)
Comment (JakeDaBoss) [reply html/changelogs/TheLion1675 - Spellcheck.yml]: I'll pull it over.

Comment (JakeDaBoss) [reply html/changelogs/TheLion1675 - Spellcheck.yml]: Fixed.

**Review by NanakoAC — APPROVED** [review]

**Pull request merged**